docs/adr/000-template.md

```diff
@@ -0,0 +1,63 @@
+# 0. Template
+
+Authors:
+
+
+## Status
+
+&lt;_Draft, Proposed, Reviewing, Accepted, Rejected, Superseded_>
```

Comment:
Those braces do tend to render incorrectly, can you just use _italics_ for that?

Also, can you just write Accepted here? Most if not all new ADRs will be proposals (PR review) and then get Accepted (when merged). But this is fine too.